fronts-client/package.json

```diff
@@ -53,28 +50,23 @@
     "fast-check": "^1.20.1",
     "fetch-mock": "^6.4.2",
     "file-loader": "^6.2.0",
-    "fork-ts-checker-webpack-plugin": "^0.5.2",
     "jest": "^29.5.0",
     "jest-environment-jsdom": "^29.5.0",
     "jsdom": "^21.1.1",
     "markdown-toc": "^1.2.0",
     "prettier": "^2.0.2",
-    "puppeteer": "^19.11.1",
+    "puppeteer": "^22.3.0",
     "react-test-renderer": "^16.13.0",
     "redux-mock-store": "^1.5.3",
     "style-loader": "^0.23.1",
-    "testcafe": "^1.1.4",
+    "testcafe": "^3.5.0",
```

Comment:
Necessary to fix some odd inconsistencies between GHA and local runs. Happily, two major version bumps did not require any changes (!)